wallet key rotation gives datanode panic -invalid memory address or nil pointer dereference

Wallet key rotation test

test - test_hot_key_rotation
branch feat_clef_ethereum_key_rotation


panic appears in datanode stderr

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1cbef65]

goroutine 36589 [running]:
code.vegaprotocol.io/vega/datanode/api.(*tradingDataServiceV2).getNodeKeyRotations(0x3?, {0x2dba380?, 0xc002c9c690?}, {0xc003b38380?, 0x100d99f?}, {{0x0, 0x0}, 0xc003b40410, 0x0, 0x1})
	/Users/paulwebb/repos/vega/datanode/api/trading_data_v2.go:2712 +0x45
code.vegaprotocol.io/vega/datanode/api.(*tradingDataServiceV2).ListKeyRotations(0xc0037c36d0?, {0x2dba380, 0xc002c9c690}, 0xc003b1c2c0)
	/Users/paulwebb/repos/vega/datanode/api/trading_data_v2.go:2700 +0x1f9
code.vegaprotocol.io/vega/protos/data-node/api/v2._TradingDataService_ListKeyRotations_Handler.func1({0x2dba380, 0xc002c9c690}, {0x2716560?, 0xc003b1c2c0})
	/Users/paulwebb/repos/vega/protos/data-node/api/v2/trading_data_grpc.pb.go:3188 +0x7b
code.vegaprotocol.io/vega/datanode/api.remoteAddrInterceptor.func1({0x2dba380, 0xc002c9c3c0}, {0x2716560, 0xc003b1c2c0}, 0xc003b3c2a0, 0xc003b2c288)
	/Users/paulwebb/repos/vega/datanode/api/server.go:248 +0x637
code.vegaprotocol.io/vega/protos/data-node/api/v2._TradingDataService_ListKeyRotations_Handler({0x288f120?, 0xc0002c0600}, {0x2dba380, 0xc002c9c3c0}, 0xc0001de000, 0xc0008b59e0)
	/Users/paulwebb/repos/vega/protos/data-node/api/v2/trading_data_grpc.pb.go:3190 +0x138
google.golang.org/grpc.(*Server).processUnaryRPC(0xc001470000, {0x2dc4b80, 0xc000185040}, 0xc0038d25a0, 0xc0008c1020, 0x3c58648, 0x0)
	/Users/paulwebb/Projects/go/pkg/mod/google.golang.org/grpc@v1.48.0/server.go:1295 +0xb2b
google.golang.org/grpc.(*Server).handleStream(0xc001470000, {0x2dc4b80, 0xc000185040}, 0xc0038d25a0, 0x0)
	/Users/paulwebb/Projects/go/pkg/mod/google.golang.org/grpc@v1.48.0/server.go:1636 +0xa2f
google.golang.org/grpc.(*Server).serveStreams.func1.2()
	/Users/paulwebb/Projects/go/pkg/mod/google.golang.org/grpc@v1.48.0/server.go:932 +0x98
created by google.golang.org/grpc.(*Server).serveStreams.func1
	/Users/paulwebb/Projects/go/pkg/mod/google.golang.org/grpc@v1.48.0/server.go:930 +0x28a
```****

--- a/datanode/api/server.go
+++ b/datanode/api/server.go
@@ -368,6 +368,7 @@ func (g *GRPCServer) Start(ctx context.Context, lis net.Listener) error {
 		stakeLinkingService:        g.stakeLinkingService,
 		eventService:               g.eventService,
 		ledgerService:              g.ledgerService,
+		keyRotationService:         g.keyRotationService,
 		ethereumKeyRotationService: g.ethereumKeyRotationService,
 		blockService:               g.blockService,
 	}

